projects/whydense/cifar-100/cifar_experiment.py

```diff
@@ -20,29 +20,30 @@
 # Original Code here:
 # https://github.com/pytorch/examples/blob/master/mnist/main.py
 
-import time
 import math
+import os
 import random
-import numpy as np
+import sys
+import time
+from collections import deque
 
+import numpy as np
 import torch
 import torch.nn as nn
+from torch.utils.data import DataLoader
+from torchvision import datasets, transforms
+
+from nupic.research.frameworks.pytorch.image_transforms import RandomNoise
+from nupic.research.frameworks.pytorch.model_utils import evaluateModel, trainModel
 from nupic.torch.modules import (
-    SparseWeights,
-    SparseWeights2d,
     Flatten,
-    KWinners2d,
     KWinners,
+    KWinners2d,
+    SparseWeights,
+    SparseWeights2d,
     rezeroWeights,
     updateBoostStrength,
```

Comment:
@lucasosouza `nupic.torch` API has changed to be pep8 compliant and these functions were renamed.
See:
 - https://github.com/numenta/nupic.torch/blob/master/nupic/torch/modules/k_winners.py#L31 
 - https://github.com/numenta/nupic.torch/blob/master/nupic/torch/modules/sparse_weights.py#L29

Reply:
You should get the latest `nupic.torch` and fix the API changes before merging

Reply:
Modified and tested with new nupic.torch, please review